English_transcripts/school/tutorials/EngShapesLab/audio/C2_sfx_ShapesLab_2_TapTheStick.txt

```diff
@@ -0,0 +1 @@
+Tap the stick to draw. Tap the brom to erase your marks.
```

Comment:
Tap the stick to draw. Tap the **broom** to erase your marks.

Reply:
Corrected